platform/admin/admin-configuration-configinstaller/src/main/java/org/codice/ddf/admin/configuration/CachedConfigData.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.configuration;
+
+import java.io.File;
+import java.util.Dictionary;
+import javax.annotation.Nullable;
+import org.codice.felix.cm.internal.ConfigurationContext;
+
+public class CachedConfigData {
+
+  private File felixFile;
+
+  // Future performance/memory improvement - replace with checksum
```

Comment:
@paouelle we said it was "possible" since the likelihood of collision is incredibly small & I ran it by @stustison and @lessarderic and the impression I got is that it would be fine to do. 

Are there problem points I'm missing?